ansible_base/resource_registry/shared_types.py

```diff
@@ -58,6 +58,10 @@ class OrganizationType(SharedResourceTypeSerializer):
     RESOURCE_TYPE = "organization"
 
     name = serializers.CharField()
+    description = serializers.CharField(
+        default="",
+        allow_blank=True,
+    )
```

Comment:
Do we also need this field on the TeamType?

Reply:
Hi @BrennanPaciorek @john-westcott-iv The issue mentions only org description, but we can go ahead and add team descriptions.